models.js

```diff
@@ -74,7 +74,20 @@ function updateVotesByArticleId(articleId, inc_votes) {
     })
 }
 
+function removeCommentById(commentId) {
+    return db.query(
+      `DELETE FROM comments
+        WHERE comment_id = $1`,
+      [commentId]
+    )
+    .then((res) => {
+    if (res.rowCount === 0) {
+        return Promise.reject({ status: 404, msg: 'Comment id not found' });
```

Comment:
A small change - I would make this error message a bit more clear by stating
'no comment with given id found'
Because the current msg sounds a bit like it's describing a bad request where they did not include the comment id